4917_Sprint/8Aeron/Design Patterns/designpatterns/.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please, remove any file that is not containing code or configuration and add the .gitignore file